src/Symfony/Component/Messenger/Bridge/Redis/Transport/Connection.php

```diff
@@ -47,6 +47,7 @@ class Connection
         'auth' => null,
         'serializer' => 1, // see \Redis::SERIALIZER_PHP,
         'sentinel_master' => null, // String, master to look for (optional, default is NULL meaning Sentinel support is disabled)
+        'redis_sentinel' => null, // String, alias for 'sentinel_master'
```

Comment:
Is this change necessary?

Reply:
Yes, otherwise this line will throw:

https://github.com/symfony/symfony/blob/49d00f66d5443ffd3b561bc43a4cc399d9716e4a/src/Symfony/Component/Messenger/Bridge/Redis/Transport/Connection.php#L232